oio/cli/container/container.py

```diff
@@ -516,8 +525,28 @@ def take_action(self, parsed_args):
             listing = acct_meta['listing']
 
         columns = ('Name', 'Bytes', 'Objects', 'Mtime')
-        return columns, ((v['name'], v['bytes'], v['objects'], v['mtime'])
-                         for v in listing)
+
+        if parsed_args.versioning:
+            columns += ('Versioning',)
+            # TODO(mbo) rework to use benefits of yield if no-paging is enabled
```

Comment:
Since there is no pressure on this new feature, I suggest we implement this before merging this pull request.